openshift/analytics-api/templates/hpa.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: {{ .Values.app.name }}
 spec:
   scaleTargetRef:
-    kind: DeploymentConfig
+    kind: Deployment
```

Comment:
Should auto-scalers be deployments? It looks like they have their own designation:

apiVersion: autoscaling/v2
kind: HorizontalPodAutoscaler